{"timestamp":"2025-10-21T15:20:38.400737Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-21T15:20:38.401200Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/fetch_and_load_data_dag.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-21T15:20:38.419316Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:38.419447Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:38.419548Z","level":"info","event":"Current task name:run_dbt","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:38.419589Z","level":"info","event":"Dag name:fetch_load_dbt_pipeline","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:38.419962Z","level":"info","event":"Tmp dir root location: /tmp","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":78}
{"timestamp":"2025-10-21T15:20:38.420879Z","level":"info","event":"Running command: ['/usr/bin/bash', '-c', 'cd /opt/airflow/dbt && dbt run --profiles-dir .']","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":88}
{"timestamp":"2025-10-21T15:20:38.429173Z","level":"info","event":"Output:","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":99}
{"timestamp":"2025-10-21T15:20:40.835239Z","level":"info","event":"\u001b[0m15:20:40  Running with dbt=1.10.13","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-21T15:20:40.837759Z","level":"info","event":"\u001b[0m15:20:40  Encountered an error:","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-21T15:20:40.837906Z","level":"info","event":"Runtime Error","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-21T15:20:40.837952Z","level":"info","event":"  Could not find profile named 'dbt_core'","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-21T15:20:42.650182Z","level":"info","event":"Command exited with return code 2","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":110}
{"timestamp":"2025-10-21T15:20:42.650741Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":972,"error_detail":[{"exc_type":"AirflowException","exc_value":"Bash command failed. The command returned a non-zero exit code 2.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/bash.py","lineno":232,"name":"execute"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-21T15:20:42.651683Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:42.651985Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:42.652082Z","level":"info","event":"Task:<Task(BashOperator): run_dbt>","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:20:42.652186Z","level":"info","event":"Failure caused by Bash command failed. The command returned a non-zero exit code 2.","logger":"task.stdout"}
